.github/workflows/go.yml

```diff
@@ -0,0 +1,31 @@
+name: Go CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      // - master
+      - master-format/linted-code 
```

Comment:
Why that branch specified as target? Maybe better run action on pull requests to any branch?